src/main/java/frc/robot/commands/AutonDriveCommand.java

```diff
@@ -27,6 +36,25 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
+    LimelightData data = limelight.getData();
+    double turnRequest;
+    double moveRequest;
+    double distance;
+
+    if (data.valid()) {
+      turnRequest = Utils.map(data.xCoord(), -Limelight.RANGE_X, Limelight.RANGE_X, -0.25, 0.25);
```

Comment:
0.25 is a magic number.

Also, I highly recommend you use the [trajectory generator](https://docs.wpilib.org/en/latest/docs/software/examples-tutorials/trajectory-tutorial/creating-following-trajectory.html#generating-the-trajectory)